intake_elasticsearch/elasticsearch_seq.py

```diff
@@ -92,5 +115,9 @@ def _get_partition(self, _):
         implement paging, known to ES as "scroll"
         https://stackoverflow.com/questions/41655913/elk-how-do-i-retrieve-more-than-10000-results-events-in-elastic-search
         """
-        results = self._run_query()
+        slice_id = None
+        slice_max = None
+        if isinstance(_, tuple):
+            slice_id, slice_max = _
```

Comment:
Combine the four lines as:
`slice_id, slice_max = _ if isinstance(_, tuple) else (None, None)` ?